src/transaction/combine.ts

```diff
@@ -1,42 +1,80 @@
 import * as _ from 'lodash'
 import binary from 'ripple-binary-codec'
-import * as utils from './utils'
 import BigNumber from 'bignumber.js'
+import {ValidationError} from '../common/errors'
 import {decodeAccountID} from 'ripple-address-codec'
 import {validate} from '../common'
 import {computeBinaryTransactionHash} from '../common/hashes'
+import {JsonObject} from 'ripple-binary-codec/dist/types/serialized-type'
+
+//The transactions should all be equal EXCEPT for the Signers field
+function validateAllSignedTransactionsAreEqual( transactions : Array<JsonObject> ) {
+  const tx = transactions[0];
+  const _txSignersTemp = tx.Signers;
+  delete tx.Signers
+  
+  transactions.forEach( _tx => {
+      const _txSignersTemp = _tx.Signers
+      delete _tx.Signers;
+      
+      if ( JSON.stringify(tx) !== JSON.stringify(_tx)) {
+          throw new ValidationError('txJSON is not the same for all signedTransactions')
+      }
+
+      _tx.Signers = _txSignersTemp
+  })
+
+  tx.Signers = _txSignersTemp
+  return transactions
```

Comment:
overall it seems like this can be re-written much simpler like so:

```javascript
function validateAllSignedTransactionsAreEqual(transactions : Array<JsonObject>) {
  if (transactions.every(tx => JSON.stringify(tx) === JSON.stringify(transactions[0])) {
    return transactions
  }
  throw new ValidationError('txJSON is not the same for all signedTransactions')
}
```

Reply:
That's much better! 

The "...object" syntax is super useful for this use case :)